code/modules/games/cards/une/une_cards.dm

```diff
@@ -0,0 +1,80 @@
+#define REDCARD	"#ff0000"
+#define YELLOWCARD	"#f9d009"
+#define GREENCARD	"#00cc00"
+#define BLUECARD	"#0000ff"
+#define BLACKCARD	"#000000"
+
+/obj/item/toy/cards/une
+	name = "deck of une cards"
+	desc = "A deck of une playing cards."
+	icon = 'icons/obj/une_cards.dmi'
+	icon_state = "deck_full"
+
+/obj/item/toy/cards/une/generate_cards()
+	for(var/i = 0; i <= 9; i++)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Red [i]", "[i]", REDCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Yellow [i]", "[i]", YELLOWCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Green [i]", "[i]", GREENCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Blue [i]", "[i]", BLUECARD)
+	for(var/i = 1; i <= 9; i++)//Second full set without a 0 card this time.
+		cards += new/obj/item/toy/singlecard/une(src, src, "Red [i]", "[i]", REDCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Yellow [i]", "[i]", YELLOWCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Green [i]", "[i]", GREENCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Blue [i]", "[i]", BLUECARD)
+	for(var/i = 0; i <= 3; i++)//Black is just a placeholder and not actually used in the coloring.
```

Comment:
`var/i = 0 to 3` is prefered, I think

Reply:
I don't really see var/i=0 to X being used anywhere else. Got anywhere else that uses this?

Reply:
It's not an official standard, but it's preferred
>implying we have official standards